app/views/collections/_show_document_list_row.html.erb

```diff
@@ -1,12 +1,10 @@
 <% id = document.id %>
+<% presenter = document.is_a?(GenericWork) ? Sufia::WorkShowPresenter.new(document, nil) : nil %>
```

Comment:
So we'll need another ticket to presenter-ify this code for a collection presenter?